Correct ENUM Names and Remove FIXME Comments

im4potato opened the pull request:
## Description

This PR changes a few ENUMs to the correct types, for example, changing a few `MENU_ENUM_LABEL_VALUE`s to `MSG`, and removes the related `FIXME` comments. It also changes a few ENUM names that referred specifically to RGUI, but were actually used in all menu drivers.

## Reviewers

@hizzlekizzle

twinaphex commented:
Hi, you probably want to join us on Crowdin from now on for any future translation stuff. We now have a crowdsourced system for this instead of just regular PRs.

twinaphex commented:
@guoyunhe Can you provide him with more details on how he can participate on Crowdin?

guoyunhe commented:
First you need a Crowdin account. Then join the project https://crowdin.com/project/retroarch

When editing the source code, you only need to fix strings in msg_hash_us.h, all other languages will be updated through Crowdin script.

im4potato commented:
@twinaphex I made a Crowdin account and joined the project, but it seems that you cannot edit the US translation from there, or make any of the other changes that I made in this PR (moved a few things around, renamed some things, removed obsolete comments).

@guoyunhe If I understand correctly, the US translation is still edited using `msg_hash_us.h` and is not available on Crowdin?

guoyunhe commented:
@im4potato it is not edited but uploaded. only when the file is changed in master branch, I can upload it to Crowdin through a script in #10358

WeedyWeedSmoker commented:
@twinaphex Hum, that's the guy who initiated the whole msg_us.h refactor that @hizzlekizzle approved last month… (https://github.com/libretro/RetroArch/pull/10460)

It was in part for that kind of cleanups that he included FIXME's all over the place… 😄

Anyway, I generally like to help backport changes made by PR's back to crowdin, to avoid things getting lost in the process…

im4potato commented:
@WeedyWeedSmoker This PR gets rid of many of the FIXMes, but I'm not sure if @twinaphex is going to merge it based on our conversation on Discord.

Ideally, I'd like to finish up the cleanup for the translation files, but I think this transition to Crowdin is making that more difficult for the time being.

WeedyWeedSmoker commented:
@im4potato Ok!

@twinaphex Please consider merging this anyway… I'm not sure what kind of discussion you and @im4potato had on Discord, but I'll sure help port any changes from this PR to Crowdin…

guoyunhe commented:
@im4potato actually you don't need to do anything specific for Crowdin in this PR. I and @WeedyWeedSmoker re-import things once the PR is merged.
